ReactApps/reacttest/src/App.css

```diff
@@ -0,0 +1,220 @@
+.App {
```

Comment:
Please split styles into different files for each component. Example : App.js - App.css, Wrapper.js - Wrapper.cs, Card.js- Card.css.